### PR TITLE
Internal class cleanup

### PR DESCRIPTION
**Pull request by cmelchior**
Preparing to add some functionality to these classes I just did a cleanup of these classes so they match our other internal classes a bit better.

- All native methods grouped at the bottom + made private where possible
- Remove outdated Javadoc
- Small style fixes

@realm/java

### CODE REVIEW

**Comment by bmunkholm** [comment]
:+1:

File: realm/realm-library/src/main/java/io/realm/internal/TableView.java [hunk 1]
```diff
@@ -22,44 +22,17 @@
...
- *
- * In this class context our interest is EmployeeView which will be inherited from this class.
- *
- * The generated class will have more specialized method to do operations on TableView.
+ * The view don't copy data from the table, but contains merely has a list of row-references into the original table
+ * with the real data.
  */
 public class TableView implements TableOrView, Closeable {
-    protected boolean DEBUG = false; //true;
+    private boolean DEBUG = false; //true;
```
Comment (zaki50) [inline realm/realm-library/src/main/java/io/realm/internal/TableView.java]: should be `static final`?

Comment (cmelchior) [inline realm/realm-library/src/main/java/io/realm/internal/TableView.java]: Yes, it should

File: realm/realm-library/src/main/java/io/realm/internal/TableView.java [hunk 2]
```diff
@@ -22,44 +22,17 @@
...
- * this will produce following classes.
- *
- * 1. Employee
- * 2. EmployeeTable
- * 3. EmployeeView.
- *
- * In this class context our interest is EmployeeView which will be inherited from this class.
- *
- * The generated class will have more specialized method to do operations on TableView.
+ * The view don't copy data from the table, but contains merely has a list of row-references into the original table
```
Comment (kneth) [inline realm/realm-library/src/main/java/io/realm/internal/TableView.java]: "don't" -> "doesn't"
"merely has" -> "merely"

**Comment by kneth** [comment]
:+1: with minor comments

**Comment by cmelchior** [comment]
retest this please

**Pull request merged**